lib/Driver/ToolChains.cpp

```diff
@@ -1191,6 +1225,20 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
       /*AddRPath=*/ shared, TC);
 }
 
+static void
+addLinkSanitizerLibArgsForWindows(const ArgList &Args,
+                                 ArgStringList &Arguments,
+                                 StringRef Sanitizer,
+                                 const ToolChain &TC,
+                                 bool shared = true
+                                 ) {
+  // Sanitizer runtime libraries requires C++.
+  Arguments.push_back("-lc++");
```

Comment:
Doesn't everything need C++, not just the sanitizers?

Reply:
This is copied across from the Darwin implementation. I haven't had a chance (yet) to test the sanitisers, but everything else works fine without the `-lc++` flag – it might not even be necessary with the sanitisers.

In general, do we want to err on the side of 'untested but might work', or 'only present if it's guaranteed to work,' considering there's a lot on Windows that won't be able to be tested properly for a while?

Reply:
My thoughts are that no one will ever remove something unnecessary, but they will add what's necessary. Therefore I'd rather err on the minimal side.